smqtk_iqr/web/classifier_service/example/.gitignore

```diff
@@ -0,0 +1 @@
+.ipynb_checkpoints
```

Comment:
This is now contained in the top-level `.gitignore` so we don't need this file anymore.